atlasdb-service/src/main/java/com/palantir/atlasdb/jackson/TableMetadataDeserializer.java

```diff
@@ -55,8 +60,35 @@ public TableMetadata deserialize(JsonParser jp, DeserializationContext ctxt) thr
         } else {
             col = deserializeNamedCols(node);
         }
+        // TODO: make defaults not duplicated with TableMetadata code.
```

Comment:
Can't we just have a static `new TableMetadata()` and read the defaults from it? Also see comment below.

Reply:
could, but that would require either a) adding setters to TableMetadata or b) adding builder, neither of which I feel particularly inclined right now to fixing... :)